listcrunch/listcrunch.py

```diff
@@ -15,9 +16,9 @@ def end_run(run):
             if run is None:
                 return
             if run[0] == run[1]:
-                subparts.append(f"{run[0]}")
+                subparts.append("{}".format(run[0]))
```

Comment:
replace f strings with format - I used format for the first one instead of `str` since I think using a literal will produce a proper unicode string under both versions, where otherwise you need a compatibility shim to get `str` on 3 and `unicode` on 2